google/cloud/google_cloud_cpp_testing.bzl

```diff
@@ -1,5 +1,6 @@
 # DO NOT EDIT -- GENERATED BY CMake -- Change the CMakeLists.txt file if needed
-google_cloud_cpp_testing_HDRS = [
+"""Automatically generated source lists for ${TARGET} - DO NOT EDIT."""
```

Comment:
Is `${TARGET}` supposed to be expanded here, or is this intended to be a placeholder? I don't see this var-style place holder in all top-level comments.

Reply:
Ooops, I can fix that.